Material.Avalonia/Material.Avalonia.csproj

```diff
@@ -41,7 +41,7 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="Avalonia" Version="0.10.15" />
+      <PackageReference Include="Avalonia" Version="0.10.16" />
```

Comment:
I think Material.Avalonia should depend on the earliest version of Avalonia possible, so as not to force users to update the Avalonia version in their projects. 
Conversely, Material.Demo should be based on the latest version of Avalonia, to keep track of changes to Avalonia and possibly find bugs that will show up with an Avalonia update.